tests/web_api/test_calculate.py

```diff
@@ -47,8 +47,7 @@ def test_responses():
         ('{"persons": {"bob": {"salary": {"invalid period": 2000 }}}}', BAD_REQUEST, 'persons/bob/salary/invalid period', 'Invalid period',),
         ('{"persons": {"bob": {"salary": {"invalid period": null }}}}', BAD_REQUEST, 'persons/bob/salary/invalid period', 'Invalid period',),
         ('{"persons": {"bob": {"basic_income": {"2017": 2000 }}}}', BAD_REQUEST, 'persons/bob/basic_income/2017', 'basic_income is only defined for months',),
-        ('{"persons": {"bob": {}}, "households": null}', OK, None, None,),
-        ('{"persons": {"bob": {"birth": {"ETERNITY": "1980-01-01"} }}}', OK, None, None,),
+        ('{"persons": {"bob": {"birth": {"ETERNITY": "1980-01-01"} }}}', BAD_REQUEST, 'households', 'is not a member of any household',),
```

Comment:
There is no such a role in the country template (and adding it would be making the template unnecessarily complex).